src/mds/Server.h

```diff
@@ -231,7 +231,6 @@ class Server {
                           std::string value,
                           file_layout_t *layout);
   void handle_set_vxattr(const MDRequestRef& mdr, CInode *cur);
```

Comment:
Can we also fix the name here:

```suggestion
  void handle_client_setvxattr(const MDRequestRef& mdr, CInode *cur);
```

So it's easy to grep for when looking at `handle_client_getvxattr`.

Reply:
Done.